ftdetect/direnv.vim

```diff
@@ -2,4 +2,4 @@
 " Author:       JINNOUCHI Yasushi <me@delphinus.dev>
 " Version:      0.2
 
-autocmd BufRead,BufNewFile .envrc setfiletype direnv
+autocmd BufRead,BufNewFile .envrc,~/.config/direnv/direnvrc,~/.direnvrc setfiletype direnv
```

Comment:
does vim support expanding environment variables? In theory a user can set XDG_CONFIG_HOME to override ~/.config to some other place.

Reply:
Yes, and it is a good point. It should not be enabled for ~/.config when users set XDG_CONFIG_HOME. I will rewrite this.

Reply:
Cool, ping me when it's done and I will merge the PR